From 8c4f3559d91c59f38f379a0754be168d2686db67 Mon Sep 17 00:00:00 2001
From: MisterZtr <andey2004586@gmail.com>
Date: Sun Jul 9 19:35:06 2023 +0300
Subject: [PATCH] remove su from no-su builds

---
 target/product/base_system.mk | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/target/product/base_system.mk b/target/product/base_system.mk
index 1126fb9034..ebbe3fd915 100644
--- a/target/product/base_system.mk
+++ b/target/product/base_system.mk
@@ -398,10 +398,10 @@ PRODUCT_PACKAGES_DEBUG := \
     unwind_reg_info \
     unwind_symbols \
 
-ifeq ($(CUSTOM_BUILD),)
-PRODUCT_PACKAGES_DEBUG += \
-    su
-endif
+#ifeq ($(CUSTOM_BUILD),)
+#PRODUCT_PACKAGES_DEBUG += \
+#    su
+#endif
 
 # The set of packages whose code can be loaded by the system server.
 PRODUCT_SYSTEM_SERVER_APPS += \
