From 848f36dcc80e4269662ee8db6718cdfb1e2498f7 Mon Sep 17 00:00:00 2001
From: MisterZtr <andey2004586@gmail.com>
Date: Sun Nov 12 15:11:14 2023 +0300
Subject: [PATCH] Remove support vndk version 28

diff --git a/base.mk b/base.mk
index fd612fc..ccc0062 100644
--- a/base.mk
+++ b/base.mk
@@ -269,5 +269,3 @@ PRODUCT_PACKAGES += \
 # Two-pane layout in Settings
 PRODUCT_PACKAGES += \
     androidx.window.extensions
-
-PRODUCT_EXTRA_VNDK_VERSIONS += 28
